fbpic/fields/fields.py

```diff
@@ -571,3 +673,15 @@ def divide_by_volume( self, fieldtype ) :
         """
         for m in range(self.Nm):
             self.interp[m].divide_by_volume( fieldtype )
+
```

Comment:
If we create a list `self.envelope_mode_numbers`, then this method can be suppressed.